module1/tasks.js

```diff
@@ -0,0 +1,189 @@
+// ==== Module#1 Basics ====
+mocha.setup('bdd');
+const expect = chai.expect;
+
+// ==== Change the capitalization of all letters in a string ====
+const changeCase = (str) => { 
+  let result = '';
+  for (let i = 0; i < str.length; i++) {
+    const char = str.charAt(i);
+    if(isUpperCase(char)) {
+      result += char.toLowerCase();
+    } else {
+      result += char.toUpperCase();
+    }
+  }
+  return result;
+};
+
+function isUpperCase(char) {
+  return char == char.toUpperCase();
```

Comment:
Please, read the difference between `==` and `===` and avoid first one till you really need it.